work-item-comments_test.go

```diff
@@ -181,6 +193,7 @@ func assertComment(t *testing.T, c *app.Comment) {
 	assert.Equal(t, "comments", c.Type)
 	assert.NotNil(t, c.ID)
 	assert.NotNil(t, c.Attributes.Body)
+	assert.NotNil(t, c.Attributes.Markup)
 	assert.NotNil(t, c.Attributes.CreatedAt)
```

Comment:
This should be `require.NotNil` because you dereference the value one line below.